HACKING.md

```diff
@@ -4,11 +4,22 @@
 
 ### Setting up the Project
 
+Before starting hacking you need a properly configured development environment. Linux and macOS are supported and used on a daily basis by the core team. System dependencies include:
+* Libev: http://software.schmorp.de/pkg/libev.html
+* Oniguruma: https://github.com/kkos/oniguruma
+* OpenSSL: https://www.openssl.org/
+* GNU Multiple Precision: https://gmplib.org/
+* Git LFS
+
+The project [`Dockerfile`](./Dockerfile) contains up-to-date system configuration instructions, as used to ship into production. It is written for the Alpine Linux distribution but is meant to be adapted to other environments such as Ubuntu, macOS+brew or others. The GitHub workflow file [`.github/workflow/ci.yml`](.github/workflow/ci.yml) also contain useful commands for Ubuntu and macOS+brew. Since ocaml.org is mostly written in OCaml, a properly configured OCaml development environment is also required. Currently, ocaml.org doesn't yet compile using OCaml 5, version 4.14 of the language is used.
+
 The `Makefile` contains many commands that can get you up and running, a typical workflow will be to clone the repository after forking it.
 
 ```
 git clone https://github.com/<username>/ocaml.org.git
 cd ocaml.org
+git lfs install
+git lfs pull
```

Comment:
Should we recommend `git lfs install` before `git clone`, then there's no need to `git lfs pull`? :thinking:

Reply:
However, for people who have already checked out without lfs, that line could save them. :see_no_evil: